src/pdfviewer/PDFDocument.cpp

```diff
@@ -2758,18 +2764,7 @@ void PDFDocument::setupMenus(bool embedded)
 /*
  * Copies to clipboard position in cm captured from the pdf viewer (w.r.t. south west corner)
  */
```

Comment:
these lines should be above callGetPosFromClick

Here they make no sense at all...